docs/versioning.md

```diff
@@ -0,0 +1,48 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+
+# Versioning
+
+`tezos-packaging` provides support for all Octez stable releases and RC candidates.
```

Comment:
:thinking: `support` on it's own seems a bit odd IMO, it kinda sounds like this repository offers technical assistance on releases.
Also, the `C` in `RC` stands for candidate, so there is duplication there :slightly_smiling_face: 

What do you think of:
```suggestion
`tezos-packaging` follows all Octez releases, both stable and candidates.
```